composer.json

```diff
@@ -16,12 +16,12 @@
         }
     ],
     "require": {
-        "php": ">=7.1.3",
-        "symfony/security-core": "^3.4|^4.4|^5.0"
+        "php": ">=7.2.5",
+        "symfony/security-core": "^4.4|^5.0"
```

Comment:
Maybe bump the symfony 5 version to ^5.2 as it's the only supported version by now

Reply:
We would be stricter than the bundle in this case. I'd leave it as is. If Symfony 4.4 works, 5.0 should work as well since they share the same feature set.

Reply:
Ok 👍